website/content/en/blog/2025-09-navigating-platform-engineering.md

```diff
@@ -0,0 +1,83 @@
+---
+title:  "Navigating Platform Engineering with the maturity model assessment"
+slug: navigating-platform-engineering
+date:   2025-09-02 01:00:00 +0000
+author: Steve Fenton
+categories:
+- Article
+tags:
+- WG Platforms
+- Community Contributions
+---
+
+The [CNCF Platform Engineering community](https://cloudnativeplatforms.com) has been working on an assessment to accompany the [Platform Engineering Maturity Model](https://cloudnativeplatforms.com/whitepapers/platform-eng-maturity-model/). It's now available as a preview for feedback, but it's crucial to understand its purpose to know what it can do for you and what you need to put in once you get your score.
+
+The assessment is a series of multiple-choice questions that takes around 20 minutes to complete. It runs in your browser and no data is shared. Once complete, you'll get a report card that shows you where you sit within the Platform Engineering Maturity Model.
```

Comment:
@techmaharaj I had to rename the file as I missed the `.md` extension!

Here's the added paragraph to set expectations on the survey.